src/ImgixTransformer.js

```diff
@@ -1,7 +1,10 @@
 import ImgixClient from './ImgixClient'
 
 const absolutePathReg = /^https?/
-const imgElementSearchReg = /(<img[^>]+>)/gi
+const imgElementReg = /(<img[^>]+>)/gi
+const imgSrcReg = /src="([^"]+)"/i
+const imgElementSegsReg = /([\s\S]*) (src="[^"]+") ([\s\S]*)/gi
```

Comment:
`imgSrcReg`, `imgElementSegsReg` and `imgElementReg` could be unified to `/(<img[^>]*)(src="([^"]+)")([^>]*>)/`, then you would be able to omit the nested function calls and the `p2.match(...)[1]`.

Reply:
How do you name `/(<img[^>]*)(src="([^"]+)")([^>]*>)/` ?

Reply:
Changed.